nemo_skills/evaluation/evaluator/__init__.py

```
@@ -13,106 +13,116 @@
 
 def get_evaluator_class(eval_type: str, config: Dict[str, Any]) -> BaseEvaluator:
     """Get evaluator instance by type."""
-    if eval_type not in EVALUATOR_CLASS_MAP:
+    if eval_type not in _EVALUATOR_CLASS_MAP_PATHS:
         raise ValueError(
             f"Evaluator class not found for type: {eval_type}.\n"
-            f"Available types with class support: {list(EVALUATOR_CLASS_MAP.keys())}\n"
-            f"All supported types: {list(EVALUATOR_MAP.keys())}"
+            f"Available types with class support: {list(_EVALUATOR_CLASS_MAP_PATHS.keys())}\n"
+            f"All supported types: {list(_EVALUATOR_MAP_PATHS.keys())}"
         )
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)